### PR TITLE
Student session location

### PR DESCRIPTION
**Pull request by ericdesj**
- Added temporary location button to the SessionDetailsFragment (notified @gregpmillr )
- Updated location seeders
- Updated sessions seeders
- Created Google Map fragment for sessions that displays session location based on session's `location_id`
- Created custom marker for Google Maps

**Screenshot**
screen shot 2017-03-26 at 8 36 22 pm

**In action**
session_location

### CODE REVIEW

File: app/src/main/java/cs/dal/krush/appFragments/SessionDetailsFragment.java [hunk 1]
```diff
@@ -55,12 +60,33 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         String title = sessionCursor.getString(sessionCursor.getColumnIndex("title"));
         String studentId = sessionCursor.getString(sessionCursor.getColumnIndex("student_id"));
         String tutorId = sessionCursor.getString(sessionCursor.getColumnIndex("tutor_id"));
+        final String locationId = sessionCursor.getString(sessionCursor.getColumnIndex("location_id"));
+        System.out.println("LOCATION ID--> " + locationId);
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/appFragments/SessionDetailsFragment.java]: can we remove this print statement, or will it be necessary for future dev?

File: app/src/main/java/cs/dal/krush/appFragments/SessionLocationFragment.java [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+     */
+    private void setAddress(String address) {
+        Geocoder geoCoder = new Geocoder(getActivity(), Locale.getDefault());
+        try {
+            List<Address> addresses = geoCoder.getFromLocationName(address, 5);
+            if (addresses.size() > 0) {
+                Double lat = (double) (addresses.get(0).getLatitude());
+                Double lon = (double) (addresses.get(0).getLongitude());
+                //Log Long and Lat
+                Log.d("lat-long", "" + lat + "......." + lon);
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/appFragments/SessionLocationFragment.java]: same thing here. We can probably remove this log statement

File: app/src/main/java/cs/dal/krush/seeders/TutoringSessionSeeder.java [hunk 3]
```diff
@@ -98,5 +98,28 @@ public static void insert(DBHelper db){
...
+
+        calendarStartTime = new GregorianCalendar();
+        calendarStartTime.set(Calendar.DAY_OF_MONTH, 3);
+        calendarStartTime.set(Calendar.HOUR_OF_DAY, 11);
+        calendarStartTime.set(Calendar.MONTH, 4);
+        calendarStartTime.set(Calendar.YEAR, 2017);
+        calendarStartTime.set(Calendar.MINUTE, 25);
+
+        calendarEndTime = new GregorianCalendar();
+        calendarEndTime.set(Calendar.DAY_OF_MONTH, 3);
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/seeders/TutoringSessionSeeder.java]: maybe we should change the day of month value to be after all the demos for this project, just so we don't have to change them at a later time. Either change DAY_OF_MONTH to 20, or change MONTH to 5. (Also change this for start time so the tutoring session starts and ends on the same date).

File: app/src/main/java/cs/dal/krush/appFragments/SessionDetailsFragment.java (hunk 1 above)
Comment (ericdesj) [inline app/src/main/java/cs/dal/krush/appFragments/SessionDetailsFragment.java]: Fixed

File: app/src/main/java/cs/dal/krush/appFragments/SessionLocationFragment.java (hunk 2 above)
Comment (ericdesj) [inline app/src/main/java/cs/dal/krush/appFragments/SessionLocationFragment.java]: Fixed

File: app/src/main/java/cs/dal/krush/seeders/TutoringSessionSeeder.java (hunk 3 above)
Comment (ericdesj) [inline app/src/main/java/cs/dal/krush/seeders/TutoringSessionSeeder.java]: I tried to implement a solution to our seeders in the SAM project, where we set the seeders to `current_date() + X` to always seed in the future. Tried something and didn't work. We can look into this for our polish. For now I changed the sessions to be a date after our demo days.

**Comment by ericdesj** [comment]
Fixes are pushed up

**Comment by Mons1Oerjan** [comment]
this PR is not like most PRs.... Approved and merged!

**Pull request merged**